centralstore/redis_store.go

```diff
@@ -984,12 +999,14 @@ func (t *tracesStore) incrementSpanCounts(ctx context.Context, conn redis.Conn,
 	})
 	defer spanInc.End()
 
-	cmds := make([]redis.Command, len(spans))
-	for i, s := range spans {
-		cmds[i] = t.incrementSpanCountsCMD(s.TraceID, s.Type)
+	commands := make([]redis.Command, 2*len(spans))
+	for i := 0; i < len(commands); i += 2 {
+		span := spans[i/2]
```

Comment:
I don't think i/2 is right -- don't you want i+1?

Reply:
This is so that each span in the spans list is looped through. For each span, there are two commands to add to the commands list. I believe `i/2` will allow me to loop through each span in the spans list

Reply:
Ah, now I see it. Thanks.